File: benchmarks/pairs/vfc-js/thenify-3-3-0/fixed.js
// Provenance: thenables/thenify  (fixed).
// repo: thenables/thenify
// commit: 0d94a24eb933bc835d568f3009f4d269c4c4c17a
// parent: aa21cfb64f47238c764d258d0faf7366a8487e36
// commit_url: https://github.com/thenables/thenify/commit/0d94a24eb933bc835d568f3009f4d269c4c4c17a
// cve: CVE-2020-7677
// license: MIT
// function: thenify
// relpath: index.js
// provenance: human
// mechanism: source_reaches_sink
// upstream_start: 2

var Promise = require('any-promise')
var assert = require('assert')

module.exports = thenify

/**
 * Turn async functions into promises
 *
 * @param {Function} fn
 * @return {Function}
 * @api public
 */

function thenify(fn, options) {
  assert(typeof fn === 'function')
  return createWrapper(fn, options)
}

File: benchmarks/pairs/vibe-py/vfapi-main-put-user/fixed.py
# Provenance: naryal2580/vfapi put_user (fixed).
# repo: naryal2580/vfapi
# commit: f36f177e1a32aa49272f4eb52f8be0af3e8f6bc3
# parent: f36f177e1a32aa49272f4eb52f8be0af3e8f6bc3
# commit_url: https://github.com/naryal2580/vfapi/blob/f36f177e1a32aa49272f4eb52f8be0af3e8f6bc3/main.py#L188
# cve: 
# license: MIT
# function: put_user
# relpath: main.py
# provenance: human
# mechanism: source_reaches_sink
# upstream_start: 219

async def nosql_return_users_from_username(username: str):
    return get_nosql_users({'username': username})
